generators/app/templates/gulp/templatesFunctions.js

```diff
@@ -0,0 +1,30 @@
+var pathModule = require('path');
+
+module.exports = function(data) {
+  data = data || {};
+  var functions = [
+    {
+      name: "revisionedPath",
+      func: function (fullPath) {
+        var path = pathModule.basename(fullPath);
+        if (data.manifest) {
+          if(!data.manifest[path]) {
+            throw new Error('File '+path+' seems to not be revisioned');
+          }
+          return pathModule.join(pathModule.dirname(fullPath),
+            data.manifest[path]);
+        } else {
+          return path;
+        }
+      }
+    },
+    {
+      name: 'assetPath',
```

Comment:
I'd rename it to `assetsPath`. Plural form is already used in a variable `ASSETS_PATH`, also the name of the folder is `assets`. Just for consistency here